.github/workflows/preview.yml

```diff
@@ -36,15 +36,22 @@ jobs:
       - name: Check PHP Version
         run: php -v
 
-      - name: Clone and update the locale submodule.
-        run: git submodule update --init --remote --recursive
+      - name: Remove placeholder locale files
+        run: rm -rf *
```

Comment:
Generally not a fan of running this command. Maybe use the explicit folder name you want to delete?

Reply:
This is explicitly being run under the locale folder to delete everything in it as it's a sub-module & the release archive from the locale repo completely replaces what would otherwise be there.

The sub-module is left in place so dev builds have locales to display without manually going through the process of grabbing the release archive and setting it up.